flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -187,6 +187,13 @@ under the License.
 			<scope>test</scope>
 			<type>test-jar</type>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-core</artifactId>
+			<version>${project.version}</version>
+			<type>test-jar</type>
```

Comment:
why do we need all these `flink-core/test-jar` changes? are they related to the PR?